server/index.js

```diff
@@ -1,31 +1,30 @@
 const express = require('express');
-const bodyParser = require('body-parser');
+const path = require('path');
+// const bodyParser = require('body-parser');
 const db = require('../database/index.js');
 const mongoose = require('mongoose');
 
-let app = express();
+const app = express();
```

Comment:
Any reason in particular to change it from a let to a const?

Reply:
I was following the airbnb lint style, but havent it up to auto lint before each commit. But I did run the lint manually on this file.